DscResources/MSFT_xWindowsUpdate/MSFT_xWindowsUpdate.psm1

```diff
@@ -174,14 +172,14 @@ function Set-TargetResource
         
     }
     
-    if ($LASTEXITCODE -eq 3010)
+    if ([bool](Get-Variable -Name 'LASTEXITCODE' -ErrorAction 'SilentlyContinue'))
```

Comment:
I am presuming that this line is to test the existence of the LastExitCode variable (e.g. has it been defined rather than just Null).

An alternate way of doing this that might be slightly cleaner is:
```powershell
Test-Path variable:LastExitCode
```

It eliminates the need for an explicit cast to bool.